### ISSUE TITLE
Update Python 3 from 3.5 to 3.6

### ISSUE THREAD

**Issue by MaxGhenis** [googledatalab/datalab#1990]
I'm trying to install a package that seems to have problems with the current Python 3, though it works with Python 2. Since the [files](https://anaconda.org/ospc/taxcalc/files) have `36` in them, I'm guessing it needs 3.6.

    !conda install -c ospc taxcalc --yes

> Fetching package metadata ...............
Solving package specifications: .
> UnsatisfiableError: The following specifications were found to be in conflict:
>   - python 3.5*
>   - taxcalc -> python >=2.7,<2.8.0a0
> Use "conda info <package>" to see the dependencies for each package.

**Comment by The-Fonz** [comment]
I'd like to see this as well, I need to use some packages that use the new format string literals introduced in Python 3.6. (They work like this: `foo='bar'; f"foo is {bar}"`, and cannot be imported from `__future__`.)

**Comment by sidnuk106** [comment]
Any idea on when we can see python 3.6 available."The fastai library uses `f'{}'` string formatting which requires python 3.6."

**Comment by gerdm** [comment]
Any progress on this? I would like to make use of Python 3.6 for the same reasons outlined above.

**Comment by acarl005** [comment]
Would also love to see this.

**Comment by tycho01** [comment]
Python 3.7(.2) would be nice, it adds Python data classes.

**Comment by pgurazada** [comment]
Yes, some expected timeline on the upgrade from 3.5 to 3.6 would be great!

**Comment by Ashwin-ks** [comment]
Any updates on this?? I would like to use python 3.7 in datalab.

**Comment by pippobaudos** [comment]
Any update on this? As it would  be helpful to many people..

**Comment by ElPapi42** [comment]
This is a must :(
